sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/DocsApiConfiguration.java

```diff
@@ -0,0 +1,53 @@
+package io.stargate.sgv2.docsapi.service;
+
+/** The configuration for the document API. */
+public interface DocsApiConfiguration {
```

Comment:
Ported as-is, but this will probably be converted to Quarkus config.

Reply:
Yes this is available via the `DocumentProperties` class now..